README.md

```diff
@@ -21,6 +21,22 @@ Use [xcaddy](https://github.com/caddyserver/xcaddy) to build Caddy with the Tail
 xcaddy build v2.7.6 --with github.com/tailscale/caddy-tailscale
 ```
 
+### Running examples
+
+Multiple example configurations are provided in the [examples directory].
+These examples expect an [auth key] to be set in the `TS_AUTHKEY` environment variable.
+All nodes registered while running these examples will be ephemeral and removed after disconnect.
```

Comment:
> will be ephemeral

I guess that depends on whether the auth key is ephemeral, right? Maybe reword like 'create [ephemeral auth key][link] for nodes to be removed after disconnect'?